Week3/Homework/Js-exercises/dogAgeCalculator.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+let calculateDogAge = function(age) {
+  let dogAge = age * 7;
```

Comment:
`const` instead of `let` here please. Also, thre seems to be a typo in the homework. Actual logic should be to have `dogAge = age / 7`. 🙂